Source/CombatExtended/CombatExtended/SecondaryDamage.cs

```diff
@@ -22,6 +22,7 @@ public DamageInfo GetDinfo(DamageInfo primaryDinfo)
         {
             var dinfo = new DamageInfo(def,
                             amount,
+                            0, //Armor Penetration
```

Comment:
This will break secondary damage, needs proper AP value

Reply:
How do I get the value? Doesn't secondary damage ignore armor, or am I misremembering something?